layers/+lang/html/packages.el

```diff
@@ -155,7 +155,7 @@
      'smartparens-mode)
    '(css-mode-hook scss-mode-hook sass-mode-hook less-css-mode-hook))
 
-  (add-hook 'web-mode-hook 'spacemacs/toggle-smartparens-off))
+  (add-hook 'web-mode-hook 'turn-off-smartparens-mode))
```

Comment:
It depends how the packages relate to each other. In this case, I think the current situation is fine. @d12frosted what do you think?